client.lua

```diff
@@ -25,16 +25,24 @@ local function pickProcess()
 	}) then
 		tasking = false
         TriggerServerEvent("qb-vineyard:server:getGrapes")
-		DeleteBlip()
+		deleteBlip()
 	else
 		QBCore.Functions.Notify(Lang:t('task.cancel_task'), 'error')
 	end
 	ClearPedTasks(cache.ped)
 end
 
+local function LoadAnim(dict)
+    while not HasAnimDictLoaded(dict) do
+        RequestAnimDict(dict)
+        Wait(1)
+    end
+end
+
 local function PickAnim()
-    lib.requestAnimDict('amb@prop_human_bum_bin@idle_a')
-    TaskPlayAnim(cache.ped, 'amb@prop_human_bum_bin@idle_a', 'idle_a', 6.0, -6.0, -1, 47, 0, 0, 0, 0)
+    local ped = cache.ped
+    LoadAnim('amb@prop_human_bum_bin@idle_a')
```

Comment:
Why switch to dis if lib.requestAnimDict already does this the wait for animation load? https://github.com/overextended/ox_lib/blob/master/imports/requestAnimDict/client.lua

Reply:
For some reason is showed previous changes which I already had been fixed in my latest commit. My most recent commit fixed that.